pyomo/contrib/doe/examples/fim_doe_tutorial.ipynb

```diff
@@ -955,21 +1479,23 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Read Heatmaps\n",
+    "### Interpreting Heatmaps\n",
     "\n",
-    "A heatmap shows the change of the objective function, a.k.a. the experimental information content, in the design region. Horizontal and vertical axes are two design variables, while the color of each grid shows the experimental information content. Taking the Fig. Reactor case - A optimality as example, A-optimality shows that the most informative region is around $C_{A0}=5.0$ M, $T=300.0$ K, while the least informative region is around $C_{A0}=1.0$ M, $T=700.0$ K."
+    "A heatmap shows the change of the objective function (the experimental information content) in the design region. \n",
```

Comment:
I think this might be helpful further up - I basically skimmed the entire heatmap section because I wasn't sure why it was important / what value it added.